### PR TITLE
Add food menus and connect to schedule

### PR DESCRIPTION
**Pull request by triplingual**
Per Kathy Azevedo comment in Slack https://code4lib.slack.com/archives/C0D9KS1EU/p1715182071842359

### CODE REVIEW

**Comment by triplingual** [comment]
Reviews requested because it's a decent amount of content and because I had to infer the ways to connect things a bit. I have tried to look it over more than once but I don't want to miss things.

Draft because I have some questions in to Kathy in Slack.

File: _data/schedule.yml [hunk 1]
```diff
@@ -118,11 +156,15 @@
...
   day2: true
   time: 12:00-13:15
+  menu: day2-menus
+  submenu: Lunch
 
 - timeImg: 12.05.png
   title: Lunch
   day3: true
   time: 12:05-13:20
+  menu: day2-menus
```
Comment (escowles) [inline _data/schedule.yml:166]: I think this should be `day3-menus`

**Comment by escowles** [comment]
@triplingual This looks good to me, other than the one day2/day3 mixup I noted above. Thanks!

**Comment by phette23** [comment]
In the past, I think we've had menus be part of other schedule events, not off their own "refreshments" events with overlapping times. So something like Day 2's 10:05am Break/Social would have a menu instead of 10:05-10:20am Break/Social 10:05AM to 12:00PM Refreshments (with menu).

I don't know if we should try to stick to that? It makes the schedule a bit more concise but is technically less accurate with regards to how long refreshments are available.

File: _data/day2-menus.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+Breakfast:
+    Options:
+        "Fresh Fruit and Berry Display": "VGN, DF, GF"
+        "Greek Yogurt": "VGT, GF"
+        "Granola (contains oats and nuts)": ""
+        "Hard-Boiled Eggs": "DF, GF"
+        "Assorted Bagels": ""
```
Comment (phette23) [inline _data/day2-menus.yml:7]: There's a bunch of tags missing on the day 2 menu, like Granola is at least vegetarian but probably vegan and dairy free, hard-boiled eggs are vegetarian, assorted bagels are vegetarian (and the spreadsheet notes there are vegan ones available). I know this stuff was missing in the spreadsheet, is it worth it for us to correct though?

File: _data/schedule.yml (hunk 1 above)
Comment (triplingual) [reply _data/schedule.yml]: Fixed

File: _data/day2-menus.yml (hunk 2 above)
Comment (triplingual) [reply _data/day2-menus.yml:7]: I'll check in with Kathy or validate against the uni dining service's info page.

**Comment by triplingual** [comment]
> In the past, I think we've had menus be part of other schedule events, not off their own "refreshments" events with overlapping times. So something like Day 2's 10:05am Break/Social would have a menu instead of 10:05-10:20am Break/Social 10:05AM to 12:00PM Refreshments (with menu).
> 
> I don't know if we should try to stick to that? It makes the schedule a bit more concise but is technically less accurate with regards to how long refreshments are available.

Yeah, this is tricky given how they want to have food available x how our sched+menus are set up. If the food's just there for a long stretch of time, it's truly not pinned to any particular schedule event. Our schedule code is structured to be quite serial and this introduces a parallel element.

File: _data/day2-menus.yml (hunk 2 above)
Comment (triplingual) [reply _data/day2-menus.yml:7]: I'll note that 2023 was to some extent this same way. 
https://2023.code4lib.org/schedule/food/day1-menus.html#Breakfast
Things like sliced fruit and coffee are not marked as vegan. There's logic in not marking as vegan something that is intuitively vegan.

Comment (triplingual) [reply _data/day2-menus.yml:7]: And yet, refreshing my short-term memory, the 2024 breakfasts note that the Fresh Fruit and Berry Display is vegan, DF, and GF.

Comment (triplingual) [reply _data/day2-menus.yml:7]: OK, so in addition to the general disclaimer about dietary requirements, I've added a disclaimer about the features being in flux.

**Comment by triplingual** [comment]
Though I just marked it as ready for review, thank you for prior reviewing, and I think it is at a point where making public is of sufficient value that the imperfections are not blockers. Happy to consider an alternate view, though.

**Review by escowles — APPROVED** [review]
👍🏻 completely agree that getting this published ASAP is more important than a perfect resolution of the schedule issues

**Pull request merged**